include/libtorrent/settings_pack.hpp

```diff
@@ -1045,8 +1048,15 @@ namespace libtorrent
 			// torrents.
 			//
 			// ``active_limit`` is a hard limit on the number of active (auto
-			// managed) torrents. This limit also applies to slow torrents.
-			//
+			// managed) torrents. This limit also applies to slow torrents and it
+			// does apply to all announcement protocols as well. If you have a
+			// large amount of seeds, only a limited amount get annonced based on
```

Comment:
seeds come in "numbers" not "amounts". i.e. s/amount/number
also: "only a limited amount get announced based on" -> "only some are announced based on"